loop-server/manifest.json

```diff
@@ -2,16 +2,30 @@
   "envs" : {
     "stage": {
       "test_types": [ "stack-check", "loadtest" ],
+      "stack_label": "loopsvrstg",
```

Comment:
it's the instance tag (verbatim) that ops has assigned to those servers.  i'm pulling it into our test manifest so that i can match test environments with specific instances